src/components/Header.tsx

```diff
@@ -0,0 +1,31 @@
+
+import * as React from 'react';
+import { Link} from 'react-scroll';
+import './../styles.css';
+
+const styles: { [key: string]: React.CSSProperties } = {
```

Comment:
Same question here of whether or not we can keep these styles back in the CSS file.